docs/Client_Libraries/0_go.md

```diff
@@ -120,11 +122,11 @@ func pushFiles() {
 ### Pull files from a remote repository
 
 ```go
-func pullFiles() {
+func pullFiles() error {

```

Comment:
Just realized that the `pullFiles` example is incomplete (missing imports, etc.).  Let me add it after this PR is merged.

Reply:
I can add them in this PR.

Reply:
QED